src/pseudopeople/noise_functions.py

```diff
@@ -356,7 +356,7 @@ def write_wrong_digits(
         digit = pd.Series(digit, index=column.index, name=column.name)
         digits.append(digit)
         noised_column = noised_column + digits[i]
-    noised_column.str.strip()
+    noised_column = noised_column.str.strip()
```

Comment:
lol whoops. 

This is like when I found one of the integration tests was passing b/c we were doing `np.isclose()`...but we weren't actually asserting anything.

Reply:
It's just False but don't worry about it XD